PWGHF/D2H/TableProducer/candidateSelectorB0ToDPiReduced.cxx

```diff
@@ -57,17 +59,34 @@ struct HfCandidateSelectorB0ToDPiReduced {
   Configurable<LabeledArray<double>> cutsDmesMl{"cutsDmesMl", {hf_cuts_ml::cuts[0], hf_cuts_ml::nBinsPt, hf_cuts_ml::nCutScores, hf_cuts_ml::labelsPt, hf_cuts_ml::labelsDmesCutScore}, "D-meson ML cuts per pT bin"};
   // QA switch
   Configurable<bool> activateQA{"activateQA", false, "Flag to enable QA histogram"};
-
+  // B0 ML inference
+  Configurable<bool> applyB0Ml{"applyB0Ml", false, "Flag to apply ML selections"};
+  Configurable<std::vector<double>> binsPtB0Ml{"binsPtB0Ml", std::vector<double>{hf_cuts_ml::vecBinsPt}, "pT bin limits for ML application"};
+  Configurable<std::vector<int>> cutDirB0Ml{"cutDirB0Ml", std::vector<int>{hf_cuts_ml::vecCutDir}, "Whether to reject score values greater or smaller than the threshold"};
+  Configurable<LabeledArray<double>> cutsB0Ml{"cutsB0Ml", {hf_cuts_ml::cuts[0], hf_cuts_ml::nBinsPt, hf_cuts_ml::nCutScores, hf_cuts_ml::labelsPt, hf_cuts_ml::labelsCutScore}, "ML selections per pT bin"};
+  Configurable<int8_t> nClassesB0Ml{"nClassesB0Ml", (int8_t)hf_cuts_ml::nCutScores, "Number of classes in ML model"};
```

Comment:
Unless I miss something, for B mesons the ML classifier should always be binary (signal and background). Hence I was wondering whether it is necessary to define a `std::vector` for it, or simply use a value. 
Maybe we could start to support a single value and in the future if there will be a usecase for multi-class classification for B mesons we can extend it.

Reply:
Currently, the way the [`MlResponse`](https://github.com/AliceO2Group/O2Physics/blob/master/Tools/ML/MlResponse.h#L60) class is written, we need vectors as input. In the config, one just needs to deactivate one of the two scores (with `cutDir = 2`) to use only one score.